ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -837,25 +855,12 @@ def set_waypoints_velocity(self):
                 self.state = 'stopped'
                 self.stopping_distance = 0.0
                 self.min_stop_distance = 0.0
-        else:
-            if (math.fabs(self.stop_calc_v -
-                    self.waypoints[self.final_waypoints_start_ptr].get_v()) > 0.1
-                    or 
-                    math.fabs(self.stop_calc_a -
-                    self.waypoints[self.final_waypoints_start_ptr].get_a()) > 0.1):
-
-                self.stop_calc_v = self.waypoints[self.final_waypoints_start_ptr].get_v()
-                self.stop_calc_a = self.waypoints[self.final_waypoints_start_ptr].get_a()
-                # self.stopping_distance = get_accel_distance(
-                #        self.waypoints[self.final_waypoints_start_ptr].get_v(),
-                #        0.0, -self.default_accel/accel_ratio,
-                #        self.waypoints[self.final_waypoints_start_ptr].get_a())
-                self.stopping_distance = self.get_min_stopping_distance(
-                        self.final_waypoints_start_ptr, -self.max_desired_jerk)
-                self.min_stop_distance = self.get_min_stopping_distance(
-                        self.final_waypoints_start_ptr, -self.max_jerk)
-
-        rospy.loginfo("dist_to_tl at ptr = {} is {:4.3f}, stopping_dist = {:4.3f}, state = "
+
+        # dont check distances when we are already trying to stop at the target
+        if ((self.state == 'slowdown' and self.stop_target == self.next_tl_wp) or\
+                self.state == 'stopped') is False:
```

Comment:
I got a fall out of state machine error when running a test where from a stopped state, I bumped the test waypoint ahead by 40 so dist_tl was outside of creep zone but since the stopping distances didn't update, the dist_tl was > min_stop_distance but < stopping_distance so it couldn't trigger a short speedup/slowdown.

If I remove this `or self.state == 'stopped'` condition, it fixes the issue since the stopping distances when stopped can be updated to near zero so dist_tl > stopping_distance threshold to allow a speedup.  Is  this condition needed to prevent some other issue?